lib/config.js

```diff
@@ -79,6 +79,7 @@ const paramsSchema = Joi.object({
   redirectUriPath: Joi.string().uri({relativeOnly: true}).optional().default('/callback'),
   required: Joi.alternatives([ Joi.function(), Joi.boolean()]).optional().default(true),
   routes: Joi.boolean().optional().default(true),
+  tokenAuthMethod: Joi.string().optional()
```

Comment:
we can add an enum here given that this abstraction over `openid-client` does not support some of the client authentication methods. I would suggest to enum `client_secret_basic` (default), `client_secret_post` and `none` to begin with.